v2_migrations/2024-08-28-081838_update_v2_primary_key_constraints/up.sql

```diff
@@ -88,12 +88,21 @@ ADD PRIMARY KEY (id);
 ------------------------ Payment Intent -----------------------
 ALTER TABLE payment_intent DROP CONSTRAINT payment_intent_pkey;
 
+-- To work around database constraints
+UPDATE payment_intent
```

Comment:
we should not do this work around, this work around will fail with all the logic.

Reply:
We can't back fill like this for older data